### PR TITLE
Update distributed-data.md

### PR DESCRIPTION
**Pull request by ZahraBayatgh**
## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

**Review by robvet — COMMENTED** [review]
Zahra, thank you so much for your input. But it looks like on line 98 you adding a space between the word 'events' and a comma. Don't space should be added there.

**Review by robvet — CHANGES_REQUESTED** [review]
Zahra, thank you so much for your input. But it looks like on line 98 you adding a space between the word 'events' and a comma. Don't think a space should be added there. Also, would not recommend changing from "read" and "write" as these terms are common across developers. 

However, really like removing the comma on line 106 following "cqrs)" and italizing on line 110.

Can you resumbit PR to match those changes.

Again, thank you so much for you suggestions!

File: docs/architecture/cloud-native/distributed-data.md [hunk 1]
```diff
@@ -95,19 +95,19 @@ A popular pattern for adding distributed transactional support is the Saga patte
 
 In the previous figure, the *Update Inventory* operation has failed in the Inventory microservice. The Saga invokes a set of compensating transactions (in red) to adjust the inventory counts, cancel the payment and the order, and return the data for each microservice back to a consistent state.
 
-Saga patterns are typically choreographed as a series of related events, or orchestrated as a set of related commands. In Chapter 4, we discussed the service aggregator pattern that would be the foundation for an orchestrated saga implementation. We also discussed eventing along with Azure Service Bus and Azure Event Grid topics that would be a foundation for a choreographed saga implementation.
+Saga patterns are typically choreographed as a series of related events , or orchestrated as a set of related commands. In Chapter 4, we discussed the service aggregator pattern that would be the foundation for an orchestrated saga implementation. We also discussed eventing along with Azure Service Bus and Azure Event Grid topics that would be a foundation for a choreographed saga implementation.
```
Comment (IEvangelist) [inline docs/architecture/cloud-native/distributed-data.md:98]:
```suggestion
Saga patterns are typically choreographed as a series of related events, or orchestrated as a set of related commands. In Chapter 4, we discussed the service aggregator pattern that would be the foundation for an orchestrated saga implementation. We also discussed eventing along with Azure Service Bus and Azure Event Grid topics that would be a foundation for a choreographed saga implementation.
```

File: docs/architecture/cloud-native/distributed-data.md [hunk 2]
```diff
@@ -95,19 +95,19 @@ A popular pattern for adding distributed transactional support is the Saga patte
...
 
 Large cloud-native applications often support high-volume data requirements. In these scenarios, traditional data storage techniques can cause bottlenecks. For complex systems that deploy on a large scale, both Command and Query Responsibility Segregation (CQRS) and Event Sourcing may improve application performance.  
 
 ### CQRS
 
-[CQRS](/azure/architecture/patterns/cqrs), is an architectural pattern that can help maximize performance, scalability, and security. The pattern separates operations that read data from those operations that write data.
+[CQRS](/azure/architecture/patterns/cqrs) is an architectural pattern that can help maximize performance, scalability, and security. The pattern separates operations that read data from those operations that write data.
 
-For normal scenarios, the same entity model and data repository object are used for *both* read and write operations.
+For normal scenarios, the same entity model and data repository object are used for *both* reading and write operations.
```
Comment (IEvangelist) [inline docs/architecture/cloud-native/distributed-data.md:108]:
```suggestion
For normal scenarios, the same entity model and data repository object are used for *both* read and write operations.
```

**Comment by IEvangelist** [comment]
Closing this PR as it's stale and the OP is unresponsive.

**Pull request closed without merging**